Code/Editor/UndoViewRotation.cpp

```diff
@@ -17,10 +17,24 @@
 // Editor
 #include "ViewManager.h"
 
+#include <AzCore/Math/Matrix3x3.h>
+#include <AzCore/Math/Matrix3x4.h>
+#include <AzFramework/Components/CameraBus.h>
+#include <MathConversion.h>
+
+Ang3 CUndoViewRotation::GetActiveCameraRotation()
+{
+    AZ::Transform activeCameraTm = AZ::Transform::CreateIdentity();
+    EBUS_EVENT_RESULT(activeCameraTm, Camera::ActiveCameraRequestBus, GetActiveCameraTransform);
+    const AZ::Matrix3x4 cameraMatrix = AZ::Matrix3x4::CreateFromTransform(activeCameraTm);
+    const Matrix33 cameraMatrixCry = AZMatrix3x3ToLYMatrix3x3(AZ::Matrix3x3::CreateFromMatrix3x4(cameraMatrix));
+    return RAD2DEG(Ang3::GetAnglesXYZ(cameraMatrixCry));
```

Comment:
nit: I think there's an `AZ::RadToDeg` function you can use instead

Reply:
Unfortunately, not in this case. RAD2DEG is a macro defined as:
```
#define RAD2DEG(a) ((a) * (180.0f / gf_PI))
```

and `Ang3` defines the necessary operators to multiply and divide by a scalar.

Reply:
Aah I see... hmm we should try and update or replace this at some point (but not right now ofc 😝)